controllers/schemas/user.js

```diff
@@ -0,0 +1,125 @@
+const validator = require('../../utils/schemaValidator')
+// const ObjectId = require('mongodb').ObjectID
+
+const getOne = {
+  id: {
+   // isMongoId: true,
+    isMongoId: true,
+    notEmpty: true,
+    errorMessage: 'This is not a correct MongoID'
+  }
+}
+
+const post = {
+  name: {
+    isAlpha: true,
+    notEmpty: true,
+    errorMessage: 'The field should contain your name with only letters'
+  },
+  office: {
+    notEmpty: true,
+    errorMessage: 'The field should contain the office name'
+  },
+  password: {
+    isAlphanumeric: true,
+    notEmpty: true,
+    errorMessage: 'Invalid password format, please use only letters and numbers'
+  },
+  email: {
+    notEmpty: true,
+    isEmail: true,
+    errorMessage: 'Invalid email'
+  },
+  phone: {
+    notEmpty: true,
+    isNumeric: true,
+    errorMessage: 'The phone is in the incorrect format'
+  },
+  role: {
+    notEmpty: true,
+    isAlpha: true,
+    errorMessage: 'You need to fullfill the field with your role'
+  },
+  slackUser: {
+    isEmail: true,
```

Comment:
slackUser is not an email it should be just a string, the goal of this field is to be able to tag a person on slag ex. @felipeNevarez